iirfilter-node/index.html

```diff
@@ -1,350 +1,416 @@
+        background-size: 60% 60%;
+        cursor: pointer;
+      }
+
+      [data-playing]:hover {
+        background-color: var(--orange);
+      }
+
+      [data-playing="true"] {
+        background: var(--green)
+          url('data:image/svg+xml;charset=utf8,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><path d="M144 479H48c-26.5 0-48-21.5-48-48V79c0-26.5 21.5-48 48-48h96c26.5 0 48 21.5 48 48v352c0 26.5-21.5 48-48 48zm304-48V79c0-26.5-21.5-48-48-48h-96c-26.5 0-48 21.5-48 48v352c0 26.5 21.5 48 48 48h96c26.5 0 48-21.5 48-48z" fill="black" /></svg>')
+          no-repeat left center;
+        background-size: 60% 60%;
+      }
+
+      /* filter button */
+      .filter-toggle {
+        margin: 4vmin auto;
+      }
+
+      .button-filter {
+        margin: 0px 0px 0px 10px;
+        padding: 0;
+        width: 90px;
+        height: 50px;
+        display: inline-block;
+        vertical-align: middle;
+        border: var(--border);
+        border-radius: 25px;
+        position: relative;
+        text-align: center;
+        transition: background 0.15s ease-in-out;
+      }
+
+      .button-filter:after {
+        content: "";
+        position: absolute;
+        height: 31px;
+        width: 31px;
+        border: var(--border);
+        border-radius: 50%;
+        background-color: var(--red);
+        top: 0px;
+        transition: left 0.15s ease-in-out;
+        will-change: left;
+      }
+
+      .button-filter[data-filteron="true"]:after {
+        background-color: var(--green);
+        left: 0px;
+      }
+
+      .button-filter[disabled] {
+        cursor: default;
+        border-color: hsla(0, 0%, 40%, 1);
+      }
+
+      .button-filter[disabled]:after {
+        background-color: hsla(342, 93%, 73%, 1);
+        border-color: hsla(0, 0%, 40%, 1);
+      }
+
+      .button-filter[data-filteron="true"][disabled]:after {
+        background-color: hsla(127, 81%, 61%, 1);
+      }
+
+      .filter-graph {
+        margin-top: 20px;
+        width: 60vw;
+        height: 40vw;
+        max-width: 600px;
+        max-height: 400px;
+      }
+    </style>
+  </head>
+  <body>
+    <div class="wrapper">
+      <div class="loading">
+        <p>Loading...</p>
+      </div>
+
+      <div class="iir-demo">
+        <button
+          class="button-play"
+          role="switch"
+          data-playing="false"
+          aria-pressed="false"
+        >
+          Play
+        </button>
+
+        <section class="filter-toggle">
+          <span id="label" aria-live="assertive" aria-atomic="true"
+            >Filter</span
+          >
+          <button
+            class="button-filter"
+            role="switch"
+            data-filteron="false"
+            aria-pressed="false"
+            aria-describedby="label"
+            disabled
+          ></button>
+        </section>
+
+        <section class="filter-graph"></section>
+      </div>
+    </div>
+
+    <script type="text/javascript">
```

Comment:
```suggestion
    <script>
```
Fix Nu HTML checker warning.